src/library/Form/Bond/BondFeedback.tsx

```diff
@@ -133,12 +133,16 @@ export const BondFeedback = ({
     }
 
     // bond amount must not be smaller than 1 planck
-    if (bond.bond !== '' && bondBn.isLessThan(1)) {
+    if (bond.bond !== '' && bondBn.isLessThan(1) && !bondBn.isNegative()) {
```

Comment:
Why doesn't if (bond.bond !== '' && bondBn.isLessThan(1)) { trigger for negative amounts ? Why do you need to explicitly add `!bondBn.isNegative()` ? 

Am I not understanding something here ?